app/assets/stylesheets/containers/show.scss

```diff
@@ -0,0 +1,27 @@
+.cityHead {
+  @extend %city-objects;
+}
+
+.reviewContainer {
+  background-color: $over-grey;
+  padding: 15px;
+  margin: 15px;
+  border-radius: 10px;
+  font-family: $fonts-sans;
+
+  h5 {
+    margin: 0px;
```

Comment:
fun fact - with the value `0` you don't have to specify any units. It's just always zero :-)